yt_idv/shaders/ray_tracing.frag.glsl

```diff
@@ -46,8 +80,18 @@ void main()
     // This will help solve the left/right edge issues.
 
     vec3 idir = 1.0/dir;
-    vec3 tl = (left_edge - camera_pos)*idir;
-    vec3 tr = (right_edge - camera_pos)*idir;
+    vec3 tl, tr, dx_cart;
+    #ifdef NONCARTESIAN_GEOM
+    tl = (left_edge_cart - camera_pos)*idir;
+    tr = (right_edge_cart - camera_pos)*idir;
+    dx_cart = right_edge_cart - left_edge_cart;
```

Comment:
This line is wrong!!!!! in the cartesian case, `dx` is grid spacing in x, y, z. Here, it should be some approximation of cartesian distance across the spherical grid (maybe just `dr`, and arc lengths along the angular distances). haven't tested if it corrects the artifacts but it's definitely a reason for why the sample factor has to be bumped so high right now.